src/javascripts/ng-admin/Crud/delete/DeleteController.js

```diff
@@ -45,14 +46,12 @@ export default class DeleteController {
             .then(() => $translate('DELETE_SUCCESS'))
             .then(text => notification.log(text, { addnCls: 'humane-flatty-success' }))
             .catch(error => {
-                const errorMessage = this.config.getErrorMessageFor(this.view, error) || 'ERROR_MESSAGE';
+                // this is kept for backward compatibility with the entity.errorMessage() method 
```

Comment:
Can you add a `@deprecated` tag? This way, when we'll ship a `2.0` version, we could `grep` all these tags to make some cleanup. :)